airbyte-integrations/connectors/source-gocardless/source_gocardless/spec.yaml

```diff
@@ -0,0 +1,45 @@
+documentationUrl: https://docsurl.com
```

Comment:
can you change to https://docs.airbyte.com/integrations/sources/gocardless ?

Reply:
Done